lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -520,10 +520,29 @@ sub schema_fixup
                 name => delete $data->{status}
             )
         }
+
+        my $packaging = delete $data->{packaging};
+        my $packaging_id = delete $data->{'packaging-id'};
+
         if ($data->{packaging}) {
```

Comment:
```suggestion
        if ($packaging) {
```
since `$data->{packaging}` is deleted above

Reply:
Fixed at https://github.com/metabrainz/musicbrainz-server/pull/1231/commits/39e40c1951b6b0a5c56c7c9be27258e937d95c4f#diff-273b5368fc014db576aec607d9a783aaR527